config/defaults.conf

```diff
@@ -54,191 +60,225 @@ local_log_path = ${run_dir}"/log.log"
 
 // Tasks //
 
-pretrain_tasks = sst // Comma-separated list of pretraining tasks or 'glue' or 'none'.
-                  // If there are multiple entries, the list should contain no spaces, and quotation marks are needed.
-                  // ex: --overrides "pretrain_tasks = \"glue,ccg\""
-target_tasks = glue  // Target tasks, in the same format as pretrain_tasks, above.
+pretrain_tasks = mnli // Comma-separated list of pretraining tasks or 'glue' or 'none'.
```

Comment:
slight preference for SST because it is small and you can finish a run very quickly, but not a big deal